Cargo.toml

```diff
@@ -14,6 +14,7 @@ description = "A command-line utility for easily compressing and decompressing f
 atty = "0.2.14"
 bstr = { version = "1.6.2", default-features = false, features = ["std"] }
 bzip2 = "0.4.4"
+bzip3 = { git = "https://github.com/bczhc/bzip3-rs", commit = "fa98dfce62d2271d0d3c91fd3592a912314a0df9", features = ["bundled"] }
```

Comment:
Alright, CI problems are blocker number 4 (I'm trying to solve it).

But I just arrived at blocker number 5: tests are failing locally.

`cargo test --release`:
```
ERROR: corrupt xz stream
```

Commenting in a piece of code to open a thread.